content/docs/glossary/claimable-balance.mdx

```diff
@@ -252,6 +307,28 @@ balances, err := client.ClaimableBalances(sdk.ClaimableBalanceRequest{Claimant:
 check(err)
 balanceId := balances.Embedded.Records[0].BalanceID
 ```
+	
+```python
+# Method 1: Not available in the Python SDK yet.
+
+# Method 2: Not available in the Python SDK yet.
```

Comment:
Is this true? Method 2 just involves parsing the transaction XDR, which I'm pretty sure is something the Python SDK must be able to do. It should map pretty directly to the other SDKs, something like

```python
tx_result = TransactionResult.from_xdr(tx_response.result_xdr)
op_results = tx_result.result.results
op_result = op_results[0].tr
if op_result.type == OperationType.CreateClaimableBalance:
    id = op_result.create_claimable_balance_result.balance_id
    id_hex = id.to_xdr_bytes().hex()
```

(untested, but you get the idea)

Reply:
Honestly, I just couldn't find how to do this in the python documentation. I'll try to fix it

Reply:
Thanks for your help! I have made the necessary changes in Method 2.